src/software/gui/standalone_simulator/widgets/standalone_simulator_gui.cpp

```diff
@@ -76,7 +76,9 @@ void StandaloneSimulatorGUI::draw()
     auto ssl_wrapper_packet = standalone_simulator->getSSLWrapperPacket();
     auto draw_function      = getDrawSSLWrapperPacketFunction(ssl_wrapper_packet);
     main_widget->simulation_graphics_view->clearAndDraw(
-        {draw_function.getDrawFunction()});
+        {main_widget->simulation_graphics_view->getDrawBallVelocityFunction()
+             .getDrawFunction(),
+         draw_function.getDrawFunction()});
```

Comment:
draw_function is a bit generic this this context, so could delete that line and do 
```suggestion
         getDrawSSLWrapperPacketFunction(ssl_wrapper_packet).getDrawFunction()});
```